js/virtual-keyboard.js

```diff
@@ -0,0 +1,213 @@
+
+import createVirtualKeyboardKeys from './create-virtual-keyboard-keys';
+
+class VirtualKeyboard {
+  constructor() {
+    this.englishLanguage = sessionStorage.getItem('englishLanguage') === 'true';
```

Comment:
It will be better if you rename this variable on languages with string property.